src/Handler/Database.hs

```diff
@@ -27,6 +27,9 @@ import qualified Language.PureScript.Docs as D
 
 import Handler.Utils
 import Handler.Caching (clearCache)
+import Data.MinLen
+
+type One = Succ Zero
```

Comment:
Building pursuit-0.4.2...
Preprocessing library pursuit-0.4.2...
[ 6 of 21] Compiling Import.NoFoundation ( src/Import/NoFoundation.hs, dist/build/Import/NoFoundation.o )
Succ and Zero are not re-exported from classy-prelude-1.0.0.1, gettinh this error:
```
Building pursuit-0.4.2...
Preprocessing library pursuit-0.4.2...
[ 6 of 21] Compiling Import.NoFoundation ( src/Import/NoFoundation.hs, dist/build/Import/NoFoundation.o )

src/Import/NoFoundation.hs:11:12:
    Not in scope: type constructor or class ‘Succ’

src/Import/NoFoundation.hs:11:17:
    Not in scope: type constructor or class ‘Zero’
```

After adding `import Data.MinLen` to `src/Import/NoFoundation.hs` getting this:
```
src/Import/NoFoundation.hs:12:1: Warning:
    Defined but not used: type constructor or class ‘One’
``